apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/logging/core/WriterFactory.java

```diff
@@ -40,4 +40,8 @@ public static IWriter getLogWriter() {
             return SystemOutWriter.INSTANCE;
         }
     }
+
+    private static boolean useSystemOut() {
+        return "system.out".equals(Config.Logging.DIR);
```

Comment:
This is not a good one. `Config.Logging.Output` should be a better choice, value options are `file` and `console`

Reply:
OK. Got it.